src/applications/_mock-form-ae-design-patterns/tests/e2e/pattern1/taskPurple.cypress.spec.js

```diff
@@ -0,0 +1,134 @@
+import manifest from '../../../manifest.json';
+import mockUsers from '../../../mocks/endpoints/user';
+import mockPrefills from '../../../mocks/endpoints/in-progress-forms/mock-form-ae-design-patterns';
+
+describe('Prefill pattern - Yellow Task', () => {
```

Comment:
todo: Change "Yellow Task" to "Purple Task"

Reply:
Whoops! thanks for catching that.